testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/BaseTestHiveCoercion.java

```diff
@@ -163,6 +163,9 @@ protected void doTestHiveCoercion(HiveTableDefinition tableDefinition)
                 "date_to_bounded_varchar",
                 "char_to_bigger_char",
                 "char_to_smaller_char",
+                "string_to_char",
```

Comment:
can you pls add a field dedicated to trimming ? 
we need to ensure that Trino functionality is compatible with Hive

Reply:
Do we need it specific to `String` as well - We have implemented one for `varchar_to_smaller_char`

Reply:
I was thinking more to have a concrete example from "abc  "  varchar(5) towards "abc" char(5)